src/Util/TestUserTrait.php

```diff
@@ -7,31 +7,65 @@
  */
 trait TestUserTrait
 {
+
     /**
      * Private create user.
      */
-    private function create($name, $pass, $roll)
+    private function createTestUsers()
+    {
+        $people = $this->getUsers();
+        foreach ($people as $person) {
+            $name = $person->name;
+            $mail = $person->mail;
+            $role = $person->role;
+            $this->taskExecStack()
+                ->stopOnFail()
+                ->exec("dktl drush user:create $name --password=$name --mail=$mail")
+                ->exec("dktl drush user-add-role $role $name")
+                ->run();
+        }
+    }
+
+    /**
+     * Get user.
+     */
+    protected function getUser($name)
```

Comment:
Could we rename this method to something like `userExists`? I found this method a bit confusing on the first read through due to the naming.